### PR TITLE
OP-731 Fix script permissions

### PR DESCRIPTION
**Pull request by mizzioisf**
- Fix permissions for oh.sh scripts
- Added comments

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -114,7 +120,9 @@ $(WIN64).zip: compile-all dw-all
 	# Set new root folder
 	sed -i 's/^\$$script\:OH_DIR\=\".\"/\$$script\:OH_DIR\=\"oh\"/g' $(WIN64)/oh.ps1
 	sed -i 's/set\ OH_DIR=\".\"/\set\ OH_DIR\=\"oh\"/g' $(WIN64)/oh.bat
+	# copy manuals
 	cp *.pdf $(WIN64)/doc
+	# create archive
```
Comment (mwithi) [inline Makefile]: is there a reason why here is "create archive" and others have "create package" ?

File: Makefile [hunk 2]
```diff
@@ -76,7 +76,11 @@ $(CLIENT).zip: compile-all
 	sed -i 's/^\$$script\:OH_MODE\=\"PORTABLE\"/\$$script\:OH_MODE\=\"CLIENT\"/g' $(CLIENT)/oh.ps1
 	sed -i 's/^\OH_MODE\=PORTABLE/OH_MODE\=CLIENT/g' $(CLIENT)/oh.sh
 	sed -i '/script:JAVA_ARCH=32/s/^#//g' $(CLIENT)/oh.ps1
+	# give exec permissions to startup script
+	chmod 755 $(CLIENT)/oh.sh
+	# copy manuals
 	cp *.pdf $(CLIENT)/doc
+	# compress package
```
Comment (mwithi) [inline Makefile]: this comment is present only here...

File: Makefile (hunk 1 above)
Comment (mizzioisf) [reply Makefile]: Right, fixed !

File: Makefile (hunk 2 above)
Comment (mizzioisf) [reply Makefile]: Right, fixed !

**Review by mwithi — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**